### ISSUE TITLE
Allow user input of filament amount when loading into AMS

### ISSUE THREAD

**Issue by misterpeppery** [bambulab/BambuStudio#414]
Probably an AMS firmware request but it would be nice to have the option to input an estimated amount of filament left when loading a spool into the AMS, and then based on slicer estimates "track" the amount of filament still left on the spool so that we can see before starting a print if there is likely enough filament left on the spool without needing to go to the printer to check.
